lib/composables/Table.ts

```diff
@@ -38,6 +38,8 @@ export interface TableColumn<V, R, SV, SR> {
   label?: string
   className?: string
   dropdown?: DropdownSection[]
+  grow?: boolean
+  fillOnAdjust?: boolean
```

Comment:
What does `fillOnAdjust` option do? 👀

Reply:
It needs better name 😅. It basically the column that expands itself to occupy rest of the space when any col is adjusted. It should be non-resizable -- usually the last column.

Reply:
Ahhh, totally make sense.

> We can probably keep it to always be the last column without need of any extra prop.

I like this idea. I guess we'll end up defining this option almost 100% the time so I think we could start by removing this option and just make it default behavior 👀